content/en/serverless/step_functions/installation.md

```diff
@@ -185,6 +185,28 @@ For sample stacks and additional code examples, see [CDK Examples for Instrument
 [3]: https://github.com/DataDog/datadog-cdk-constructs
 [4]: /serverless/guide/step_functions_cdk
 {{% /tab %}}
+{{% tab "AWS SAM" %}}
+1. Ensure you have deployed the [Datadog Lambda Forwarder][1], and that you are using v3.130.0+. You might need to [update your Forwarder][1].
```

Comment:
When would the user need to update the Forwarder?

Reply:
This line is copied from other tabs. I think it means user need to update the forwarder if the version is too old.

Reply:
actually, it does seem a bit redundant to say. I'll remove that line entirely.